google-login-plugin/src/com/google/gct/login/stats/GctTracking.java

```diff
@@ -23,13 +23,14 @@
   public static final String CATEGORY = "com.google.cloud.tools";
 
   // actions
-  public static final String NEW_ENDPOINT = "new.endpoint";
   public static final String DEPLOY = "appengine.update";
-  public static final String VCS = "vcs";
   public static final String GRADLE_IMPORT = "appengine.gradle.import";
-  public static final String RUN = "appengine.run";
-  public static final String WIZARD = "appengine.wizard";
+  public static final String LOGIN = "user.login";
```

Comment:
yeah I think that would be the right thing.

On Mon, Oct 19, 2015 at 1:19 PM, nbashirbello <notifications@github.com>
wrote:

> In google-login-plugin/src/com/google/gct/login/stats/GctTracking.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/132#discussion_r42399047>
> :
>
> >    public static final String GRADLE_IMPORT = "appengine.gradle.import";
> > -  public static final String RUN = "appengine.run";
> > -  public static final String WIZARD = "appengine.wizard";
> > +  public static final String LOGIN = "user.login";
>
> @patflynn <https://github.com/patflynn> I moved this class to the login
> plugin so that I could add the LOGIN tracking string, since the core plugin
> depends on the login plugin. I could move the LOGIN tracking string in
> another file, maybe called LoginTracking and put GctTracking back in the
> core plugin. Would this be better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/132/files#r42399047>
> .
>